docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -63,14 +63,37 @@ Set the Elasticsearch settings before deploying the manifest:
   value: "https://somesuperhostiduuid.europe-west1.gcp.cloud.es.io:443"
 ------------------------------------------------
 
+// Begin collapsed section
+[%collapsible]
+.Configuration details
+====
+****
+
+[cols="2*<a"]
+|===
+| Setting | Description
+
+include::configuration/env/shared-env.asciidoc[tag=elasticsearch-username]
+
+include::configuration/env/shared-env.asciidoc[tag=elasticsearch-password]
+
+include::configuration/env/shared-env.asciidoc[tag=elasticsearch-host]
+
+|===
+
+See <<agent-environment-variables>> for all available options.
+
+****
+====
+
 [discrete]
-=== Running {agent} on master nodes
+=== Run {agent} on master nodes
 
-Kubernetes master nodes use https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/[taints]
+Kubernetes master nodes can use https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/[taints]
 to limit the workloads that can run on them. The manifest for standalone {agent} defines
-tolerations to run on master nodes, which collects metrics from the control plane
-components of Kuberentes (scheduler, controller manager).
-To disable {agent} from running on master nodes remove the following part of the Daemonset spec:
+tolerations to run on master nodes. These tolerations collect metrics from the control plane
```

Comment:
```suggestion
tolerations to run on master nodes. Agents running on master nodes collect metrics from the control plane
```

Tolerations are used to make it possible for Agents' Pods to run on Master nodes.